### DISCUSSION TITLE
Can depth maps be trimmed?

### DISCUSSION THREAD

**Discussion by ghost** [cdcseacave/openMVS#874] [Q&A] [answered]
Sorry for the terrible question, is it possible to trim depth maps to roi before fusion?

When processing scenes with an roi, more time and memory was used to fuse a large chunk only to be trimmed down.
Could that be useful for tileable processing of a scene? Less memory usage per tile at the cost of reloading all depth maps and saving to disk in one instance or using sub scene split with the added benefit of resuming processing.

It would be a better alternative to the current sub scene with complete dense clouds and no overlap for any dataset type.

**Comment by cdcseacave** [comment, marked as answer]
Yes, it is possible to trim depth-maps instead, which would be fairly easy to implement, though not sure how much that would help (ie the depth-map image will use the same space regardless if it stores true depths or only 0s)
